solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,15 @@
+package ru.sberbank.school.task02;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+
+  @Override
+  public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    return new Converter(externalQuotesService);
+  }
```

Comment:
&#39;method def rcurly&#39; has incorrect indentation level 2, expected level should be 4.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1361516716>*